src/pages/index.js

```diff
@@ -100,3 +193,38 @@ const addCardFormValidator = new FormValidator(
 
 editProfileFormValidator.enableValidation();
 addCardFormValidator.enableValidation();
+
+function handleDeleteCard(cardId, cardElement) {
+  deleteCardPopup.open();
+  deleteCardPopup.handleDeleteConfirm(() => {
+    deleteCardPopup.renderLoading(true);
+    api
+      .deleteCard(cardId)
+      .then(() => {
+        cardElement.removeCard();
+        deleteCardPopup.close();
+      })
+      .catch(console.error)
+      .finally(() => {
+        deleteCardPopup.renderLoading(false);
+      });
+  });
+}
+
+function handleLikeCard(cardId, cardElement) {
+  if (!cardElement._isLiked) {
```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.